pom.xml

```diff
@@ -6,7 +6,7 @@
   <parent>
     <groupId>org.apache</groupId>
     <artifactId>apache</artifactId>
-    <version>24</version>
+    <version>31</version>
   </parent>
 
   <groupId>org.apache.shirp</groupId>
```

Comment:
Why do we have "shirp" rather than "shiro" in the `groupId`?

Reply:
It seems to be an old error :(

Reply:
Oops! Luckily just the site.

Reply:
this is now fixed